src/init-action.ts

```
@@ -98,7 +98,7 @@ async function run() {
 
     const workflowErrors = await actionsUtil.getWorkflowErrors();
 
-    if (workflowErrors.length > 0) {
+    if (workflowErrors.filter(o => o.code !== 'LintFailed').length > 0) {
```

This will mean we hide the warning if we fail to link the workflow. I thought we wanted to show it?

---

I've added a comment. I don't think we want to warn people visually but we do want to report the error via status.